hazelcast/predicate.py

```diff
@@ -586,7 +589,7 @@ def not_(predicate):
     return _NotPredicate(predicate)
 
 
-def between(attribute, from_, to):
+def between(attribute: str, from_: typing.Any, to: typing.Any) -> Predicate:
```

Comment:
I think `typing.Any`s can be safely left out. There are more throughout the code.

Reply:
See my comment above. It is done on purpose